README.md

```diff
@@ -88,7 +88,7 @@ we will need the following configuration information from our CA:
     For `step-issuer`, we will Base64-encode this PEM block:
 
     ```sh
-    $ CA_ROOT_B64=$(kubectl get -o jsonpath="{.data['root_ca\.crt']}" configmaps/step-certificates-certs | base64)
+    $ CA_ROOT_B64=$(kubectl get -o jsonpath="{.data['root_ca\.crt']}" configmaps/step-certificates-certs | base64 -w 0)
```

Comment:
The wrapping option only works on the GNU version of base64. It is not present on macOS or BSD-based distributions because they do not wrap. Instead of using this option we can use `step base64` that will work everywhere:

```suggestion
    $ CA_ROOT_B64=$(kubectl get -o jsonpath="{.data['root_ca\.crt']}" configmaps/step-certificates-certs | step base64)
```